### ISSUE TITLE
Unspecified error [304bd60258]

### ISSUE THREAD

**Issue by aem-intellij-plugin-error-reporter** [aem-intellij-plugin-error-reporter/auto-generated-issues-storage#64]
fixed:0.11.3

## Description
*No description*
## Exception
```
java.lang.ClassCastException: class com.jetbrains.plugins.jade.psi.impl.JadeFakeXmlNameElement cannot be cast to class com.intellij.psi.xml.XmlToken (com.jetbrains.plugins.jade.psi.impl.JadeFakeXmlNameElement is in unnamed module of loader com.intellij.ide.plugins.cl.PluginClassLoader @53a7d564; com.intellij.psi.xml.XmlToken is in unnamed module of loader com.intellij.util.lang.UrlClassLoader @51565ec2)
	at co.nums.intellij.aem.bb.b(Unknown Source)
	at co.nums.intellij.aem.eA.a(Unknown Source)
	at co.nums.intellij.aem.eA.processQuery(Unknown Source)
	at com.intellij.openapi.application.QueryExecutorBase.execute(QueryExecutorBase.java:77)
	at com.intellij.util.ExecutorsQuery.processResults(ExecutorsQuery.java:31)
	at com.intellij.util.AbstractQuery.doProcessResults(AbstractQuery.java:97)
	at com.intellij.util.AbstractQuery.delegateProcessResults(AbstractQuery.java:120)
	at com.intellij.util.MergeQuery.processResults(MergeQuery.java:26)
	at com.intellij.util.AbstractQuery.doProcessResults(AbstractQuery.java:97)
	at com.intellij.util.AbstractQuery.delegateProcessResults(AbstractQuery.java:120)
	at com.intellij.util.UniqueResultsQuery.processResults(UniqueResultsQuery.java:38)
	at com.intellij.util.AbstractQuery.doProcessResults(AbstractQuery.java:97)
	at com.intellij.util.AbstractQuery.forEach(AbstractQuery.java:89)
	at com.intellij.util.AbstractQuery.findAll(AbstractQuery.java:30)
	at com.intellij.codeInsight.daemon.impl.IdentifierHighlighterPass.getUsages(IdentifierHighlighterPass.java:174)
	at com.intellij.codeInsight.daemon.impl.IdentifierHighlighterPass.getHighlightUsages(IdentifierHighlighterPass.java:150)
	at com.intellij.codeInsight.daemon.impl.IdentifierHighlighterPass.lambda$highlightTargetUsages$0(IdentifierHighlighterPass.java:207)
	at com.intellij.util.AstLoadingFilter.disallowTreeLoading(AstLoadingFilter.java:125)
	at com.intellij.codeInsight.daemon.impl.IdentifierHighlighterPass.highlightTargetUsages(IdentifierHighlighterPass.java:206)
	at com.intellij.codeInsight.daemon.impl.IdentifierHighlighterPass.doCollectInformation(IdentifierHighlighterPass.java:98)
	at com.intellij.codeHighlighting.TextEditorHighlightingPass.collectInformation(TextEditorHighlightingPass.java:55)
	at com.intellij.codeInsight.daemon.impl.PassExecutorService$ScheduledPass.lambda$null$1(PassExecutorService.java:429)
	at com.intellij.openapi.application.impl.ApplicationImpl.tryRunReadAction(ApplicationImpl.java:997)
	at com.intellij.codeInsight.daemon.impl.PassExecutorService$ScheduledPass.lambda$doRun$2(PassExecutorService.java:422)
	at com.intellij.openapi.progress.impl.CoreProgressManager.registerIndicatorAndRun(CoreProgressManager.java:585)
	at com.intellij.openapi.progress.impl.CoreProgressManager.executeProcessUnderProgress(CoreProgressManager.java:531)
	at com.intellij.openapi.progress.impl.ProgressManagerImpl.executeProcessUnderProgress(ProgressManagerImpl.java:59)
	at com.intellij.codeInsight.daemon.impl.PassExecutorService$ScheduledPass.doRun(PassExecutorService.java:421)
	at com.intellij.codeInsight.daemon.impl.PassExecutorService$ScheduledPass.lambda$run$0(PassExecutorService.java:397)
	at com.intellij.openapi.application.impl.ReadMostlyRWLock.executeByImpatientReader(ReadMostlyRWLock.java:167)
	at com.intellij.openapi.application.impl.ApplicationImpl.executeByImpatientReader(ApplicationImpl.java:157)
	at com.intellij.codeInsight.daemon.impl.PassExecutorService$ScheduledPass.run(PassExecutorService.java:395)
	at com.intellij.concurrency.JobLauncherImpl$VoidForkJoinTask$1.exec(JobLauncherImpl.java:161)
	at java.base/java.util.concurrent.ForkJoinTask.doExec(ForkJoinTask.java:290)
	at java.base/java.util.concurrent.ForkJoinPool$WorkQueue.topLevelExec(ForkJoinPool.java:1020)
	at java.base/java.util.concurrent.ForkJoinPool.scan(ForkJoinPool.java:1656)
	at java.base/java.util.concurrent.ForkJoinPool.runWorker(ForkJoinPool.java:1594)
	at java.base/java.util.concurrent.ForkJoinWorkerThread.run(ForkJoinWorkerThread.java:177)
```
## Error context
```
Last Action:       Git.Pull
Plugin Version:    0.11.2
OS Name:           Mac OS X
Java Version:      11.0.4
Java VM Vendor:    JetBrains s.r.o
Product Name:      WebStorm
Full Product Name: WebStorm
App Version Name:  WebStorm
Is EAP:            false
App Build:         WS-193.5233.80
App Version:       2019.3
```

**Comment by aem-intellij-plugin-error-reporter** [comment]
## Description
*No description*
## Error context
```
Last Action:       null
Plugin Version:    0.9.1
OS Name:           Windows 10
Java Version:      11.0.5
Java VM Vendor:    JetBrains s.r.o
Product Name:      PyCharm
Full Product Name: PyCharm
App Version Name:  PyCharm
Is EAP:            false
App Build:         PY-193.5662.61
App Version:       2019.3.1
```

**Comment by aem-intellij-plugin-error-reporter** [comment]
## Description
*No description*
## Error context
```
Last Action:       null
Plugin Version:    0.8.5
OS Name:           Linux
Java Version:      1.8.0_212
Java VM Vendor:    SAP AG
Product Name:      IDEA
Full Product Name: IntelliJ IDEA
App Version Name:  IntelliJ IDEA
Is EAP:            false
App Build:         IU-182.4323.46
App Version:       2018.2.3
```

**Comment by karollewandowski** [comment]
Thank you for reporting it.
Fixed in 0.11.3, please update the plugin.
